share/governance.md

```diff
@@ -0,0 +1,95 @@
+
+* advocating for use of the project
+* informing developers of project strengths and weaknesses from a new user’s perspective
+* providing moral support (a ‘thank you’ goes a long way)
+* writing documentation and tutorials
+* filing bug reports and feature requests
+* participating on the discussion board
+
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become contributors, as described below.
+
+*How to become one*: Use a Vox Pupuli project
+
+
+Collaborators
+-------------
+
+
+Collaborators are community members who submit patches to the project. These patches may be a one-time occurrence or occur over time. Expectations are that contributors will submit patches that are small at first and will only grow larger once the contributor has built confidence in the quality of their patches.
+
+*How to become one*: Submit a pull request to a Vox Pupuli project
+
+
+Contributors
+------------
+
+
+Contributors are collaborators who have shown wide dedication to the Vox Pupuli project in general or deep dedication to one project in particular, and the ability to work well with collaborators and users. The contributors have responsibilities beyond the collaborators. In particular, contributors formally decide on whether a pull request is merged or a release is cut. Contributors are recognized and valued for their contributions, in whatever form those contributions take. Tasks such as communication, documentation, art, packaging, and testing are all valued contributions. A contributor will use lazy consensus to decide on whether to merge a pull request from a collaborator. If the discussion is no longer moving towards a consensus, the PMC must vote via lazy consensus on whether the patch should be applied.
+
+*How to become one*: Be a collaborator and be nominated to the PMC as a contributor. Nominations should be sent to <email address for nominations>. You may nominate yourself. It is also common to give contributor status to an individual who donates code to the project by migrating a repository to the github namespace.
+
+
+Project Management Committee (PMC)
+----------------------------------
+
+
+The project management committee has responsibilities beyond contributors that include participating in strategic planning, release planning, and approving changes to the governance model. They also have the duty of upholding the community code of conduct and ensuring its values. The PMC is also responsible for updating the community code of conduct. It also makes decisions when community consensus cannot be reached. 
```

Comment:
The reality is we're all pretty new to this governance stuff, and the first year of leadership will be eye-opening for everyone. Giving the PMC the authority to modify the governance document gives them the power to move quickly. This is a representative democracy with year-long terms, the authority of the PMC is derived from votes by the governed.

I agree that it would be better if the governance document could only be modified by a majority vote in a poll of all community members. But I don't think that's reasonable. I think, because of our own inexperience drafting this kind of stuff and leading communities in general, and because Vox Pupuli is something of a new model, we're gonna need to make a number of governance changes right out of the gate.

If I thought I could write a perfect governance document, or this was me in 10 years with more experience, I would not give the PMC the power to rewrite it, and I would make them go out to the community and get every change ratified. I don't think that's the case. I worry that direct-democracy is hard to do (we're already talking about doing it once a year) and that voter fatigue in open source communities is very high. We're specifically talking about enfranchising the users of our software, which is a large body of humans who often don't care at ALL about governance in open source communities.

After a couple years, when the governance document and election process have 'settled', I'm all for changing the document so that the PMC has to go out to the voting population to get major changes ratified.